test/client/test.js

```diff
@@ -34,32 +34,141 @@ requirejs(['qunit', 'jquery', 'auth'], function(QUnit, $, auth) {
     });
 
     QUnit.test('Data is loaded into table on page load', function(assert) {
-      assert.expect(3);
+      assert.expect(5);
 
       var oldAjaxGet = $.get;
       var data = [
         {
           token: 'abc',
           status: 'valid',
-          user: 'user@example.com'
-        }
+          user: 'user@example.com',
+        },
+        {
+          token: 'xyz',
+          status: 'revoked',
+          user: 'ruser@example.com',
+        },
       ];
-      $.get = function mockAjax(url, cback) {
-        assert.strictEqual(url, '/listTokens',
+
+      $.get = function mockAjax(opts) {
+        assert.strictEqual(opts.url, '/listTokens',
           'Makes request to /listTokens');
-        cback(data, 'success');
-        assert.strictEqual(
-          $('#token-table').children('tbody').children('tr').length, 2,
+        opts.success(data, 'success');
+
+        var tokenTableRows = $('#token-table').children('tbody').children('tr');
+        assert.strictEqual(tokenTableRows.length, 3,
           'Table is not empty after request to /listTokens'
         );
+        assert.notOk(tokenTableRows.filter('tr:nth-child(2)').hasClass('disabled-row'));
+        assert.ok(tokenTableRows.filter('tr:nth-child(3)').hasClass('disabled-row'));
       };
+
       assert.strictEqual(
         $('#token-table').children('tbody').children('tr').length, 1,
         'Table is empty before request to /listTokens'
       );
       auth.setupPage();
       $.get = oldAjaxGet;
     });
+
+    QUnit.test('Show error message', function(assert) {
```

Comment:
Missing test for error returning from ajax mocked call. To see the flow is complete. Error is displayed when ajax returns error.